worlds/dkc3/Regions.py

```diff
@@ -924,9 +923,6 @@ def create_region(world: MultiWorld, player: int, active_locations, name: str, l
 
                 location = DKC3Location(player, locationName, loc_id, ret, loc_byte, loc_bit, loc_invert)
                 ret.locations.append(location)
-    if exits:
-        for exit in exits:
-            ret.exits.append(Entrance(player, exit, ret))
```

Comment:
Removing this, but leaving the `exits=None` in the function is probably not a good idea, should this ever get used.

Reply:
the reason i didn't do it here is because every call of this function has a hardcoded `exits=None`